### ISSUE TITLE
Version property name not being read and property name option does not work.

### ISSUE THREAD

**Issue by chrischappell-rgare** [skarpdev/dotnet-version-cli#113]
I am getting no output when reading the version of a project. The version was originally incremented using version 2.4.1.0.

dotnet version
```
dotnet-version-cli version 3.0.1.0
Project version is:
```

If I rename the Version element to PackageVersion in the csproj I get the version back:
```
dotnet-version-cli version 3.0.1.0
Project version is:
     1.0.0
```
 If I run version again with the version property name override set to Version I still see the version coming back even though the csproj is still using PackageVersion instead of Version.
 
 dotnet version -v Version
 ```
dotnet-version-cli version 3.0.1.0
Project version is:
     1.0.0
```

**Comment by nover** [comment]
Hi,
It would seem like a bug has snuck in with the latest updates to the handling of multiple version fields. I'll see if I can find some time during the Easter break to debug it.

**Comment by nover** [comment]
Closed via #115
